eu.stethewwolf.gresistor.json

```diff
@@ -0,0 +1,31 @@
+{
+    "app-id": "eu.stethewwolf.gresistor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "gresistor3",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=x11"
```

Comment:
it's either `x11` or `fallback-x11` + `wayland`

(unless there is a good reason to have both x11 and wayland at the same time)